### PR TITLE
Add brunch

### PR DESCRIPTION
**Pull request by sos-alena**

### CODE REVIEW

**Comment by tatianaburii** [comment]
Hello)
I want to choose my action, but computer does it for me :(

**Comment by sos-alena** [comment]
Hello!! The era of artificial intelligence has arrived!

вт, 15 нояб. 2022 г. в 15:59, Tetiana Burii ***@***.***>:

**Comment by sos-alena** [comment]
Added the ability for the player to make their own move

File: src/main/java/myGames/Game2.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    static String myGame2(String playerOne, String playerTwo) {
+        String result = null;
+
+        System.out.println("The " + playerOne + " makes a move");
+        String itemOne = chooseItem();
+        System.out.println("The " + playerTwo + " makes a move");
+        String itemTwo = chooseItem();
+
+        if (itemOne.equals("Stone") && itemTwo.equals("Scissors")) {
```
Comment (kolodgeeva) [inline src/main/java/myGames/Game2.java]: I think this big if-else can be optimized with some Map))

File: src/main/java/myGames/Utils.java [hunk 2]
```diff
@@ -68,4 +68,20 @@ public static int oneOrZero() throws IOException {
             return oneOrZero();
         }
     }
+    public static String chooseItem() {
+        try {
+            String item = READER.readLine();
+            if (item.equals("Stone") || item.equals("Paper") || item.equals("Scissors")){
```
Comment (kolodgeeva) [inline src/main/java/myGames/Utils.java]: For Stone/Paper/Scissors strings we can create Enum, because a lot of repetitions

Comment (sos-alena) [reply src/main/java/myGames/Utils.java]: Ok, I will do it!

File: src/main/java/myGames/Game2.java (hunk 1 above)
Comment (sos-alena) [reply src/main/java/myGames/Game2.java]: Ok, thanks, something to think about))

File: src/main/java/myGames/Game.java [hunk 3]
```diff
@@ -58,52 +59,29 @@ private static void absoluteWinner(String playerOne, String playerTwo, int One,
...
-        }
+    static String myGame(String playerOne, String playerTwo) {
+        String result;
+
+        System.out.println("The " + playerOne + " makes a move");
+        Items itemOne = chooseItem();
+        System.out.println("The " + playerTwo + " makes a move");
+        Items itemTwo = chooseItem();
+
+        HashMap<ArrayList<Items>, String> play = new HashMap<>();
```
Comment (kolodgeeva) [inline src/main/java/myGames/Game.java]: Now it's better.
We can move the map to a static field
And we can improve key, for example, it's can be String PAPER_STONE

Comment (kolodgeeva) [reply src/main/java/myGames/Game.java]: or Pair
https://www.baeldung.com/java-pairs

Comment (kolodgeeva) [reply src/main/java/myGames/Game.java]: actually you can merge, I just trying to give you ideas how we can improve the solution)

Comment (sos-alena) [reply src/main/java/myGames/Game.java]: Ok, Thanks!!!

Comment (kolodgeeva) [reply src/main/java/myGames/Game.java]: Looks good)

**Pull request merged**